### ISSUE TITLE
How to install this using Docker 

### ISSUE THREAD

**Issue by bryceprutsos** [hihouhou/huginn_epic_store_new_free_game_agent#2]
I am trying to add this agent but having some issues 

from my env file I know is being used
ADDITIONAL_GEMS=huginn_epic_store_new_free_game_agent

from logs

web_1        | Fetching gem metadata from https://rubygems.org/.......
web_1        | Could not find gem 'huginn_epic_store_new_free_game_agent' in rubygems
web_1        | repository https://rubygems.org/ or installed locally.
threaded_1   | Fetching gem metadata from https://rubygems.org/.......
threaded_1   | Could not find gem 'huginn_epic_store_new_free_game_agent' in rubygems
threaded_1   | repository https://rubygems.org/ or installed locally.

from the lib dir: ~/docker_volumes/huginn/huginn/lib$
drwxrwxr-x 2 X X 4096 Nov 27 02:39 huginn_epic_store_new_free_game_agent
-rw-rw-r-- 1 X X 159832 Nov 27 02:38 huginn_epic_store_new_free_game_agent.rb

pwd: ~/docker_volumes/huginn/huginn/lib/huginn_epic_store_new_free_game_agent
-rw-rw-r-- 1 x x 242777 Nov 27 02:39 epic_store_new_free_game_agent.rb

I thought I had all the files in place any ideas?

**Comment by hihouhou** [comment]
try: 
`ADDITIONAL_GEMS=huginn_epic_store_new_free_game_agent(github: hihouhou/huginn_epic_store_new_free_game_agent)`

**Comment by bryceprutsos** [comment]
Thanks, @hihouhou, that worked perfectly.

One more maybe enhancement for this is to have memory on it like Huginn RSS feeds. 
I had it running once an hour and it would send the current games (Star Wars) every hour. I have since changed it to every Monday so that shouldn't be such an issue. The memory would definitely help out with that. PS I have installed a few more of your agents thanks for your work.
